docs/pages/admin-guides/management/guides/awsoidc-integration.mdx

```diff
@@ -17,7 +17,7 @@ The following features use an AWS OIDC integration to interact with AWS:
 
 It targets users who would prefer a more manual approach or to manage the integration with Infrastructure as Code tools.
 
-As an alternative to this guide, you can use the Teleport Web UI (Access Management / Enroll New Integration).
+As an alternative to this guide, you can use the Teleport Web UI (Add New / Integration).
```

Comment:
Would it be possible to make it more explicit that the text in the parentheses is a sequence of UI clicks?